packages/base-map/src/index.story.tsx

```diff
@@ -140,3 +146,22 @@ export const optionalLayers = () => (
 customLocationPopupContent.parameters = a11yOverrideParameters;
 
 export const onContextMenuPopup = () => <ContextMenuDemo />;
+
+/**
+ * See https://visgl.github.io/react-map-gl/docs/api-reference/attribution-control
+ * See https://visgl.github.io/react-map-gl/docs/api-reference/navigation-control
+ * See https://visgl.github.io/react-map-gl/docs/api-reference/scale-control
+ *
+ * Any control which is added as a child of a react-map-gl map is supported
+ */
+export const withOptionalControls = () => (
+  <BaseMap
+    mapLibreProps={{ attributionControl: false }}
```

Comment:
Insert comment explaining why adding a `attributionControl: false`?

Reply:
Sort props

Reply:
Sorted and comment added in 82cf6e389b83c136c2d8370515fdbcfead3294b6